Source/UltraleapTrackingCore/Public/LeapTrackingSettings.h

```diff
@@ -0,0 +1,33 @@
+/******************************************************************************
+ * Copyright (C) Ultraleap, Inc. 2011-2024.                                   *
+ *                                                                            *
+ * Use subject to the terms of the Apache License 2.0 available at            *
+ * http://www.apache.org/licenses/LICENSE-2.0, or another agreement           *
+ * between Ultraleap and you, your company or other organization.             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/NoExportTypes.h"
+#include "LeapUtility.h"
+#include "LeapTrackingSettings.generated.h"
+
+
+/**
+ * TODO All settings should be added here 
+ */
+UCLASS(config = Game, defaultconfig)
+class ULeapTrackingSettings : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	UPROPERTY(config, EditAnywhere, Category = "Ultraleap Settings")
+	TArray<FString> UltraleapHints;
```

Comment:
How does it pick up the name of the settings? Does it turn this member into "Ultraleap Hints"? I need to check this code in editor.

Reply:
This will show up in the editor under plugins->ultraleaptracking->UltraleapHints